docs/advanced-features/self-signed-certificates.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+To use a self-signed certificate with the `quilt3` API or CLI,
+the certificate must first be added to your Python environment.
+
+## Mac and Linux
+
+1. Determine the location of your certificates:
+    ```python
+    >>> import certifi
+    >>> certifi.where()
+    '/path/to/site-packages/certifi/cacert.pem'
+    ```
+2. Copy your custom self-signed certificate and paste into the
+`cacert.pem` file.
```

Comment:
This is actually not a good solution cause that will will be overwritten any time `certifi` gets upgraded. Setting ``REQUESTS_CA_BUNDLE` is better.

On Linux, need set `REQUESTS_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt`.

On OS X, it turns out it's more complicated - need to first create the bundle file. Assuming you're using OS X), can you see if this works? https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/non-standard-certs.html

And then on both Linux and OS X, `export REQUESTS_CA_BUNDLE=...` should be added to `~/.bashrc`.